_includes/markdown/JavaScript.md

```diff
@@ -123,6 +123,80 @@ console.log( d ); // 4
 
 Use destructuring whenever possible to slim down your code and improve overall readability.
 
+#### Spread operator and Rest parameters
+The notation `...` which comes with ES6 represents the **spread** operator and **rest** parameters.  
```

Comment:
@mserino can we focus this more so on the "why" and "when" for using the spread operator. Something like answering the question or "When is it a best practice to use this?" For code samples and such we can just link off site to MDN or something